src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -3805,6 +3808,472 @@ .maxstack  1
 }");
         }
 
+        [Fact]
+        [WorkItem(31665, "https://github.com/dotnet/roslyn/issues/31665")]
+        public void TestQuestionableSequencePoints_31665()
```

Comment:
>Questionable [](start = 24, length = 12)

Consider removing this qualifier from the test names, to avoid implying the expected sequence points are questionable.